README.md

```diff
@@ -1 +1,26 @@
 # hw01
+
+## Biography of David Kanos
+
+I was born on *April 24th, 1996* in **Jos, Nigeria**  I am currently getting my Masters in Public Policy at **The University of Chicago.**
+
+### Family
+
+I have two younger sisteres named *Elsa* and *Natalia* and I love them both very much. My mum runs and NGO in Nigeria that helps widows and orphans affected by Boko Haram in Nigeria. My dad is an investor and consoultant in Nigeria. 
```

Comment:
Hi, David. Like Yuwen, I also noticed some typos such as "sisteres" and "consoultant." They definitely don't really matter much for this assignment, though some could be confusing in the future.

Reply:
On that note, RStudio has [a built-in spell checker](https://blog.rstudio.com/2020/02/11/rstudio-1-3-preview-real-time-spellchecking/) that can be helpful when proofreading documents